src/www/app/outline_server.ts

```diff
@@ -57,6 +60,11 @@ export class OutlineServer implements PersistentServer {
     return this.config.host;
   }
 
+  get usesUnsupportedCipher() {
```

Comment:
Let's add this to the Server model. It can be a a method. I think it makes it clearer it's calculated.

Reply:
Ack